serve/serve.py

```diff
@@ -48,15 +48,15 @@ class WrapperHandler(object):
     def __init__(self, base_path=None, url_base="/"):
         self.base_path = base_path
         self.url_base = url_base
-        self.handler = handlers.handler(self.handle_request)
```

Comment:
Why remove this? All it does is printing a nice traceback on the 500 error page if there's an exception.